libr/asm/arch/arm/aarch64/aarch64-opc.c

```diff
@@ -2256,9 +2255,9 @@ print_register_offset_address (char *buf, size_t size,
   if (print_extend_p)
     {
       if (print_amount_p)
-	snprintf (tb, tblen, ",%s %d", shift_name, opnd->shifter.amount); // #
+	snprintf (tb, 16, ",%s %d", shift_name, opnd->shifter.amount); // #
```

Comment:
use sizeof (tblen) instead of hardcoded 16, this way if we have to change the size of the array we will only have to do it in one place